File: src/PhoneTicket/PhoneTicket.Android/src/phoneticket/android/model/Location.java
package phoneticket.android.model;

public class Location implements ILocation {
	public double longitude;
	public double latitude;

	public Location(double longitude, double latitude) {
		this.longitude = longitude;
		this.latitude = latitude;
	}

	@Override
	public double getLongitude() {
		return longitude;
	}

	@Override
	public double getLatitude() {
		return latitude;
	}

}

File: trunk/src/PhoneTicket.Web/PhoneTicket.Web/Models/Location.cs
﻿namespace PhoneTicket.Web.Models
{
    using System.Globalization;

    public class Location
    {
        private const string GeoFormat = "POINT({0} {1})";

        public double Latitude { get; set; }

        public double Longitude { get; set; }

        public override string ToString()
        {
            return string.Format(
                GeoFormat,
                this.Longitude.ToString(CultureInfo.InvariantCulture),
                this.Latitude.ToString(CultureInfo.InvariantCulture));
        }
    }
}